### PR TITLE
basemaps config in app_config.yml

### PR DESCRIPTION
**Pull request by javisantana**
@xavijam @javierarce I removed the hardcoded tiles in the css, I don't understand why we included in the css instead of adding dynamically from javascript

@juanignaciosl please, could you review the backend code ? does central need to change?

doc: https://github.com/CartoDB/cartodb/wiki/How-to-configure-basemaps-in-CartoDB

before merging, this other PR should be merged: https://github.com/CartoDB/cartodb-cookbooks/pull/246

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1325/))

**Comment by javisantana** [comment]
cc @santisaez  @azamorano hey, this should be enough to insert whatever basemaps you want to include in onpremise

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1326/))

**Comment by juanignaciosl** [comment]
:+1: 

Central will have some useless code, but since CartoDB slices allowed attributes (`self.set(params.slice(*allowed_attributes_from_central(action)))`) there should be no need for change.

File: app/assets/stylesheets/map/dropdowns/basemap_dropdown.css.scss [hunk 1]
```diff
@@ -227,16 +216,6 @@ div.dropdown.basemap {
...
-      &.nokia_reduced_day .thumb     { background-image: url("https://1.maps.nlp.nokia.com/maptile/2.1/maptile/newest/reduced.day/6/30/24/256/png?token=A7tBPacePg9Mj_zghvKt9Q&app_id=KuYppsdXZznpffJsKT24"); } 
-      &.nokia_night .thumb           { background-image: url("https://1.maps.nlp.nokia.com/maptile/2.1/maptile/newest/normal.night/6/30/24/256/png?token=A7tBPacePg9Mj_zghvKt9Q&app_id=KuYppsdXZznpffJsKT24"); }
-      &.nokia_terrain_day .thumb     { background-image: url("https://1.maps.nlp.nokia.com/maptile/2.1/maptile/newest/terrain.day/6/30/24/256/png?token=A7tBPacePg9Mj_zghvKt9Q&app_id=KuYppsdXZznpffJsKT24"); }
-      &.nokia_satellite_day .thumb   { background-image: url("https://1.maps.nlp.nokia.com/maptile/2.1/maptile/newest/satellite.day/6/30/24/256/png?token=A7tBPacePg9Mj_zghvKt9Q&app_id=KuYppsdXZznpffJsKT24"); }
-      &.nokia_hybrid_day .thumb      { background-image: url("https://1.maps.nlp.nokia.com/maptile/2.1/maptile/newest/hybrid.day/6/30/24/256/png?token=A7tBPacePg9Mj_zghvKt9Q&app_id=KuYppsdXZznpffJsKT24"); }
-
```
Comment (xavijam) [inline app/assets/stylesheets/map/dropdowns/basemap_dropdown.css.scss]: :+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1373/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1376/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1377/))

**Pull request merged**